scripts/collect-stats.ts

```diff
@@ -75,6 +92,19 @@ class StatsCollector {
         if (!reviewedPRs[login].has(pull_request_url)) {
           contributors[login].reviews[repo.name] =
             (contributors[login].reviews[repo.name] || 0) + 1;
+
+          if (!contributors[login].reviews_by_category[repo.name]) {
+            contributors[login].reviews_by_category[repo.name] = {};
+          }
+
+          for (const key of Object.keys(customCategories)) {
```

Comment:
You can use a [for...in loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) for this!
```suggestion
          for (const key in customCategories) {
```
I’d also recommend using something more descriptive than key given how we use it. For example, we could rename it to `categoryName`.